dataland-frontend/tests/e2e/specs/landing-page/LandingPage.ts

```diff
@@ -0,0 +1,29 @@
+describe("Check that the Landing Page to work properly", () => {
```

Comment:
This test currently does not run in the CI because it is not included in the index.ts of the folder it is located in.